.github/workflows/continuous-deployment.yml

```diff
@@ -48,6 +48,75 @@ jobs:
         with:
           args: "--skip-sorting-imports --replace"
 
+  accessibility-test:
+    defaults:
+      run:
+        working-directory: user-interface
+
+    runs-on: ubuntu-latest
+
+    environment: ${{ needs.Build-Info.outputs.environment }}
```

Comment:
We don't have to do it here, but I've been thinking we probably can remove this line. I like the move up too, I had realized the accessibility-test job was in a weird place after it got merged.